liiatools/datasets/annex_a/lds_annexa_clean/converters.py

```diff
@@ -9,6 +9,8 @@ def to_integer(value):
     :param value: Some value to convert to an integer
     :return: Either an integer value or a blank string
     """
+    if isinstance(value, str) and value[-2:] == ".0":
```

Comment:
Should this deal with strings containing more than a single decimal place?

Reply:
This will only convert strings with one decimal place. I could try and make it look for the decimal place and allow any number of trailing zeros afterwards but I haven't ever seen a file be read in this way so I think it will make it unnecessarily complicated. This script reads in csv files and from testing .000 with different number formats (text, general, number) it seems to handle all of those fine

Reply:
Actually forgot I changed this for Annex A which does read in .xlsx and so if for some reason someone is saving a column as "text" with .00 then it won't work